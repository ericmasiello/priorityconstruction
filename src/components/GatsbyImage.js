import Img from 'gatsby-image';
import Image from './Image';

export default Image.withComponent(Img);
