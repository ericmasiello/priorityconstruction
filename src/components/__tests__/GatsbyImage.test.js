import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import GatsbyImage from '../GatsbyImage';

it('should render <GatsbyImage>', () => {
  const component = renderer.create(<GatsbyImage sizes={{ src: '/foo/bar.jpg' }} />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
