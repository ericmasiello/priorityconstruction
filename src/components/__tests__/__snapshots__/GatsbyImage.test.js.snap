// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <GatsbyImage> 1`] = `
.c0 {
  max-width: 100%;
}

<div
  className=" gatsby-image-outer-wrapper"
  style={
    Object {
      "position": "relative",
    }
  }
>
  <div
    className="c0 gatsby-image-wrapper"
    style={
      Object {
        "overflow": "hidden",
        "position": "relative",
      }
    }
  >
    <div
      style={
        Object {
          "paddingBottom": "NaN%",
          "width": "100%",
        }
      }
    />
    <img
      alt=""
      onLoad={[Function]}
      src="/foo/bar.jpg"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "objectFit": "cover",
          "objectPosition": "center",
          "opacity": 1,
          "position": "absolute",
          "top": 0,
          "transition": "opacity 0.5s",
          "width": "100%",
        }
      }
    />
    <noscript
      dangerouslySetInnerHTML={
        Object {
          "__html": "<img src=\\"/foo/bar.jpg\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:0.5s;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/>",
        }
      }
    />
  </div>
</div>
`;
